### PR TITLE
Testing Pull Request

### PR DESCRIPTION
**Pull request by Redjumpman**
Just testing what I can add to a PR

### CODE REVIEW

File: casino/casino.py [hunk 1]
```diff
@@ -23,6 +23,8 @@
 class Casino:
     """Casino"""
 
+    __slots__ = ["bot", "legacy_path", "legacy_system", "legacy_available", "file_path", "system", "games", "deck", "card_values", "version"]
```
Comment (Chovin) [inline casino/casino.py]: comments and such

File: heist/heist.py [hunk 2]
```diff
@@ -92,8 +92,8 @@ def __init__(self, bot):
         elif not self.heist_started(settings):  # Checks if a heist is currently happening
             await self.bot.say("You can't join an ongoing heist")
         elif self.heist_plan(settings):  # checks if a heist is being planned or not
-            self.heist_ptoggle(settings)
-            self.crew_add(user.id, user.name, settings)
+            await self.heist_ptoggle(settings)
```
Comment (Chovin) [inline heist/heist.py]: and this one is a code review or some such

File: heist/heist.py [hunk 3]
```diff
@@ -154,26 +154,26 @@ def __init__(self, bot):
                 t = tabulate(z, headers=["Players", "Credits Stolen", "Bonuses", "Total Haul"])
                 await self.bot.say("The total haul was split among the winners:\n```Python\n{}```".format(t))
                 settings["Config"]["Time Remaining"] = int(time.perf_counter())
-                self.heistclear(settings)
+                await self.heistclear(settings)
```
Comment (Chovin) [inline heist/heist.py]: booop boop

File: casino/casino.py [hunk 4]
```diff
@@ -89,8 +96,8 @@ def __init__(self, bot):
                         transfer = {user.id: old_data}
                         settings["Players"].update(transfer)
                         self.legacy_system["Players"].pop(user.id)
-                        dataIO.save_json(self.legacy_path, self.legacy_system)
-                        dataIO.save_json(self.file_path, self.system)
+                        await dataIO.flush_json(self.legacy_path, self.legacy_system)
```
Comment (Chovin) [inline casino/casino.py]: multi-file code review

File: casino/casino.py [hunk 5]
```diff
@@ -294,13 +301,13 @@ def __init__(self, bot):
                                 await self.bot.say("The coin flips into the air...")
                                 await asyncio.sleep(2)
                                 settings["Players"][user.id]["Played"]["Coin Played"] += 1
-                                dataIO.save_json(self.file_path, self.system)
+                                await dataIO.flush_json(self.file_path, self.system)
```
Comment (Chovin) [inline casino/casino.py]: poop

File: casino/casino.py [hunk 6]
```diff
@@ -258,12 +265,12 @@ def __init__(self, bot):
...
                                 await asyncio.sleep(3)
                                 settings["Players"][user.id]["Played"]["Cups Played"] += 1
-                                dataIO.save_json(self.file_path, self.system)
+                                await dataIO.flush_json(self.file_path, self.system)
                                 if cup == outcome:
                                     amount = bet * settings["Games"]["Cups"]["Multiplier"]
                                     await self.bot.say("Congratulations! The coin was under cup {}!".format(outcome))
                                     settings["Players"][user.id]["Won"]["Cups Won"] += 1
-                                    dataIO.save_json(self.file_path, self.system)
+                                    await dataIO.flush_json(self.file_path, self.system)
```
Comment (Chovin) [inline casino/casino.py]: a

**Comment by Kowlin** [comment]
having fun there...

**Comment by Twentysix26** [comment]
🚮

**Pull request closed without merging**